### DISCUSSION TITLE
Translation Support?

### DISCUSSION THREAD

**Discussion by dybdeskarphet** [claudiodangelis/qr-filetransfer#232] [Ideas]
I'm opening this issue because:

- [ ] I have found a bug
- [x] I want to request a feature
- [x] I have a question
- [ ] Other

Hello,
I really like qrcp and I want it to be more popular around the world and I'm hoping to do it by translating, are you planning to add translation (po files) support in the future?

**Comment by claudiodangelis** [comment]
Hello @dybdeskarphet, this is actually an interesting proposal.
I had a quick look around and it seems that this package may help managing the internationalization of the program: https://github.com/nicksnyder/go-i18n

However, it does not seem to support PO files. I will have a look in the next weeks.

Thanks,
Claudio
